source/ScrollSync/ScrollSync.example.js

```diff
@@ -236,8 +236,12 @@ export default class GridExample extends React.PureComponent {
   _renderLeftSideCell({columnIndex, key, rowIndex, style}) {
     const rowClass =
       rowIndex % 2 === 0
-        ? columnIndex % 2 === 0 ? styles.evenRow : styles.oddRow
-        : columnIndex % 2 !== 0 ? styles.evenRow : styles.oddRow;
+        ? columnIndex % 2 === 0
```

Comment:
Why are these changed?

Reply:
```
239:32  error  Replace `·?·styles.evenRow` with `⏎··········?·styles.evenRow⏎·········`  prettier/prettier
  240:32  error  Replace `·?·styles.evenRow` with `⏎··········?·styles.evenRow⏎·········`  prettier/prettier
```

Reply:
Because these codes would cause eslint errors so I fixed it 0.0